recipe/meta.yaml

```
@@ -1,42 +1,61 @@
-{% set version = "1.6.0rc1" %}
+{% set name = "bluesky" %}
+{% set version = "1.5.4" %}
+{% set sha256 = "309d284bb3f95539e1d89a76b4e640f59a6b575308582690c9fbc259a9cd329c" %}
 
 package:
-    name: bluesky
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    git_url: https://github.com/nsls-ii/bluesky
-    git_rev: v{{ version }}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/NSLS-II/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-    number: 0
-    script: python setup.py install --single-version-externally-managed --record=record.txt
-    skip: True  # [py2k]
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-    build:
-        - python
-
-    run:
-        - cycler
-        - databroker
-        - event-model >=1.10.0
-        - historydict
-        - ipython
-        - lmfit
-        - matplotlib
-        - numpy
-        - ophyd
-        - python
-        - pyzmq
-        - super_state_machine
-        - toolz
-        - tqdm
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cycler
+    - databroker
+    - doct
+    - event-model
```

I think we still should use the pinning of the `event-model` as it's done in our [original recipe](https://github.com/NSLS-II/lightsource2-recipes/blob/899ac4866fab7e3a9f72b6bc499b627b44e9841f/recipes-tag/bluesky/meta.yaml#L23) (conda-forge's recipe may have missed that update):
```suggestion
    - event-model >=1.10.0
```